src/index.html

```diff
@@ -23,6 +23,7 @@
     </div>
 </div>
 
+
```

Comment:
This extra line break isn't needed. Only a single line between blocks is necessary.